appdatabase/migrations/sql/1702651464_community_mailservers.up.sql

```diff
@@ -0,0 +1,13 @@
+-- To allow many to many relationship between communities and mailservers
+CREATE TABLE IF NOT EXISTS community_mailservers (
+    community_id BLOB NOT NULL,
+    mailserver_id VARCHAR NOT NULL,
+    PRIMARY KEY (community_id, mailserver_id),
+    FOREIGN KEY (community_id) REFERENCES communities_communities(id) ON DELETE CASCADE,
+    FOREIGN KEY (mailserver_id) REFERENCES mailservers(id) ON DELETE CASCADE
+) WITHOUT ROWID;
+
+CREATE INDEX community_mailservers_mailserver_id ON community_mailservers(mailserver_id);
+
+-- To avoid querying community-only mailservers
+ALTER TABLE mailservers ADD COLUMN community_only BOOLEAN DEFAULT FALSE;
```

Comment:
I'd use `storenodes` instead of `mailservers`, as mailserver is a concept associated to wakuv1 (eventually we'll have to rename everywhere we use mailservers to storenodes to be consistent)

Reply:
it is an existing table so I rather don't change the table name

Reply:
`mailservers` is an already existing table, I rather leave that name